src/sentry/integrations/slack/integration.py

```diff
@@ -53,11 +54,24 @@ def get_pipeline_views(self):
 
         return [identity_pipeline_view]
 
+    def get_team_info(self, access_token):
+        payload = {
+            'token': access_token,
+        }
+
+        session = http.build_session()
+        resp = session.post('https://slack.com/api/team.info', data=payload)
```

Comment:
I like that this is a POST and not a GET. Going to blindly trust that this is correct.

Reply:
I do enjoy that `POST` works.. but looking at the docs it does say "Preferred method `GET`". 2480fb0